doc/storage.md

```diff
@@ -0,0 +1,77 @@
+# Storage
+
+The piraeus operator can automate some basic storage set up for LINSTOR. This includes:
+
+* Preparing physical devices. [Read LINSTOR guide](https://www.linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-a_storage_pool_per_backend_device)
+* Creating storage pools. [Read LINSTOR guide](https://www.linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-storage_pools)
+
+## Preparing physical devices
+
+The piraeus operator can automatically configure devices on satellites. Depending on the value of
+`operator.nodeSet.automaticStorageType`, all _empty_ devices will be added to a storage pool (one pool per device name,
```

Comment:
Mention that the size must be > 1GiB. It is really `>` and not `>=`, oddly enough.